tox.ini

```diff
@@ -43,22 +64,27 @@ deps =
     sunpy: tqdm
     sunpy: zeep
 extras =
-    dev
-commands =
+    tests
+commands_pre =
+    oldestdeps: minimum_dependencies drms --filename requirements-min.txt
+    oldestdeps: pip install -r requirements-min.txt
     pip freeze --all --no-input
-    sunpy: pytest -vvv -s -ra --pyargs sunpy.net.jsoc --timeout=120 --remote-data=any {posargs}
```

Comment:
@nabobalis is there a good reason why this was in commands_pre? Do you want `-sunpy` to run both sunpy and drms tests?

Reply:
There is no good reason for this being here.

I was just lazy.